rdkit/Chem/Draw/SimilarityMaps.py

```diff
@@ -181,8 +181,14 @@ def GetSimilarityMapFromWeights(mol, weights, colorMap=None, scale=-1, size=(250
       if cm is not None and isinstance(colorMap, type(cm.Blues)):
         # it's a matplotlib colormap:
         clrs = [tuple(x) for x in colorMap([0, 0.5, 1])]
-      else:
+      elif isinstance(colorMap, list):
+        # assume it's a passed in list of colors
         clrs = [colorMap[0], colorMap[1], colorMap[2]]
+      else:
+        # assume it's a matplotlib colormap ID
+        customCmap = cm.get_cmap(colorMap, 3)
+        clrs = [customCmap(0), customCmap(1), customCmap(2)]
```

Comment:
The last piece of this is going to fail with an unhappy error message if `cm=None` and the whole thing should also work if the colors are in a tuple instead of a list. So how about something like this (untested):
```
    if cm is not None:
      if isinstance(colorMap, type(cm.Blues)):
        # it's a matplotlib colormap:
        clrs = [tuple(x) for x in colorMap([0, 0.5, 1])]
      else:
        # assume it's a matplotlib colormap ID
        customCmap = cm.get_cmap(colorMap, 3)
        clrs = [customCmap(0), customCmap(1), customCmap(2)]
    elif isinstance(colorMap, list) or isinstance(colorMap,tuple):
      # assume those are colors
      clrs = [colorMap[0], colorMap[1], colorMap[2]]
    else:
      raise ValueError("colorMap arguments that are not lists or tuples can only be used if matplotlib is installed.")
```